ReleaseNotes.md

```diff
@@ -6,6 +6,12 @@
 ## Highlights
 
 ## Backwards incompatible changes
+All methods returning `JSMol` and `JSReaction` objects now return a `nullptr`
```

Comment:
Is all this stuff actually part of this PR?

Reply:
Nope, removed.